lib/github-package.js

```diff
@@ -263,6 +370,7 @@ export default class GithubPackage {
 
     ReactDom.render(
       <RootController
+        pipelineManager={this.pipelineManager}
```

Comment:
Since the pipelines live in the Repo now, we don't need to pass these down through the JSX, right?

Reply:
oh yeah, good catch 👍